validate.go

```diff
@@ -144,6 +144,11 @@ func checkEventHookPaths(hookType string, hooks []rspec.Hook, hooksCheck bool) {
 			if fi.Mode()&0111 == 0 {
 				logrus.Fatalf("The %s hook %v: is not executable", hookType, hook.Path)
 			}
+			for _, env := range hook.Env {
+				if !envValid(env) {
+					logrus.Fatalf("Env %q for hook %v is in the invalid form.", env, hook.Path)
+				}
+			}
```

Comment:
This check is host-independent, so it should happen regardless of whether `--hooks` is set (and that portability was the main reason for the flag in #49.  For example, see [here][1]).

[1]: https://github.com/opencontainers/ocitools/pull/49#issuecomment-216585596